packages/opencensus-core/src/stats/model/view.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Aggregation, Measure, Tags, View, ViewPath} from './types';
+
+export class StatsView implements View {
+  private viewName: string;
+  private path = {} as ViewPath;
+  description: string;
+  measure: Measure;
+  columns: Tags;
+  aggregation: Aggregation;
+
+  constructor(
+      name: string, description: string, measure: Measure,
+      aggregation: Aggregation, columns: Tags) {
+    this.name = name;
+    this.description = description;
+    this.measure = measure;
+    this.aggregation = aggregation;
+    this.columns = columns;
+  }
+
+  /**
+   * Sets the view name and the path
+   */
+  set name(name: string) {
+    const folders = name.split('/');
+    let nextPath = this.path;
```

Comment:
If you set the name twice (say `a/b` and then `c/d`), we will just end up getting a really long path `a -> b -> c -> d`. Is this intended?

Reply:
set name() removed.